docs/src/main/asciidoc/transaction-guide.adoc

```diff
@@ -113,6 +113,13 @@ public class SantaClausService {
 <1> Inject the `TransactionManager` to be able to activate `setRollbackOnly` semantic.
 <2> Programmatically decide to set the transaction for rollback.
 
+
+=== Transaction Configuration
+Extended configuration of the transaction is possible with the use of the `@TransactionConfiguration` annotation that needs to be set in addition to the standard `@Transactionnal` annotation (at method or class level).
```

Comment:
s/that needs to be set in addition/that is set in addition/

It conveys the same meaning but less aggressive.

Reply:
done